docs/installation.md

```diff
@@ -56,21 +56,21 @@ helm template openfeature/open-feature-operator -s templates/{CRD} --set default
 
 ### Uninstall
 ```sh
-helm uninstall ofo
+helm uninstall openfeature
```

Comment:
Can you perhaps make one small addition to specify that `openfeature` is the name of the release (in the install command).